app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -212,7 +218,7 @@ fun HikeDetails(
     savedHikesViewModel: SavedHikesViewModel,
     elevationData: List<Double>
 ) {
-  val hikeDetailState by savedHikesViewModel.hikeDetailState.collectAsState(null)
+  val hikeDetailState = savedHikesViewModel.hikeDetailState.collectAsState(null).value
```

Comment:
Question: What is the benefit of this change?

Reply:
It was a fix about an unused import reported by Sonar